timon/plugins/trio.py

```diff
@@ -1,6 +1,8 @@
+import logging
```

Comment:
this is no more a plugin.
A plugin is optional. trio is now compulsory. move module one level up.

Reply:
Moved and renamed